history/index.d.ts

```diff
@@ -1,18 +1,21 @@
-// Type definitions for history v2.0.0
-// Project: https://github.com/rackt/history
-// Definitions by: Sergey Buturlakin <https://github.com/sergey-buturlakin>, Nathan Brown <https://github.com/ngbrown>
+// Type definitions for history v3.0.0
+// Project: https://github.com/ReactTraining/history
+// Definitions by: Sergey Buturlakin <https://github.com/sergey-buturlakin>, Nathan Brown <https://github.com/ngbrown>, Viacheslav Slinko <https://github.com/vslinko>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-// types based on https://github.com/rackt/history/blob/master/docs/Terms.md
+// types based on https://github.com/ReactTraining/history/blob/master/docs/Glossary.md
 
-export as namespace History;
+export as namespace ReactTrainingHistory;
```

Comment:
According to TypeScript docs `export as namespace` should be used to declare name of a global variable of module.
It means we can't change it, because `history` is exposed as `History` global variable: https://github.com/mjackson/history/blob/master/webpack.config.js#L5